nni/algorithms/compression/pytorch/pruning/admm_pruner.py

```diff
@@ -113,7 +114,9 @@ def _projection(self, weight, sparsity, wrapper):
         tensor
             the projected matrix
         '''
-        return weight.data.mul(self.masker.calc_mask(sparsity, wrapper)['weight_mask'])
+        wrapper_copy = copy.deepcopy(wrapper)
```

Comment:
Why deepcopy here?

Reply:
It uses copied wrapper as a temporary variable to calculate mask for `weight` tensor to fix the problem that Guoxin mentioned.